pudb/debugger.py

```diff
@@ -2953,6 +2962,11 @@ def make_frame_ui(i, frame_lineno):
 
     def update_cmdline_win(self):
         self.set_cmdline_state(not CONFIG["hide_cmdline_win"])
+        
+    def update_header(self, override=None):
+        if override is not None:
+            self.top._w.header = self.header if override else None
+        self.top._w.header = self.header if not CONFIG["hide_header"] else None
```

Comment:
Is there an `else` missing here? Otherwise, it would seem the `override` logic would be ineffective. (Why is it needed anyhow?)

Reply:
Actually, there's a return missing here between these lines. Good catch. I knew something was off with it.

The `override` is just a glorified `show_header` function that I squished into a single function. In hindsight there's no reason to not just have a separate function for it. The purpose was for showing exceptions since the warning for it occurs in the header and it needs to be displayed even if the header is hidden.

I'll break it up into two clearer functions and you can see how you feel about it